pkg/cinderbackup/statefulset.go

```diff
@@ -62,30 +57,33 @@ func StatefulSet(
 	}
 
 	args := []string{"-c"}
+	// The liveness probe is disabled when debugging the service, but the HTTP
+	// liveness checker server is always started.
```

Comment:
We could leave the probe container out when the CR has debug set for the service, but it's not really important.